Streetcode/Streetcode.WebApi/Program.cs

```diff
@@ -31,6 +33,7 @@
 
 app.UseCors();
 app.UseHttpsRedirection();
+app.UseMiddleware<ApiRequestResponseMiddleware>();
```

Comment:
```suggestion
app.UseRequestResponseMiddleware();
```
This should look like this